src/plone/restapi/tests/test_batching.py

```diff
@@ -172,10 +174,11 @@ def test_contains_correct_batch_of_items(self):
         response = self.api_session.get("/collection?b_start=2&b_size=2")
 
         # Response should contain second batch of items
-        self.assertEqual(
-            [u"/plone/folder/doc-2", u"/plone/folder/doc-3"],
-            result_paths(response.json()),
+        _result_paths = result_paths(response.json())
+        self.assertTrue(
+            all(path.startswith(u"/plone/folder/doc-") for path in _result_paths)
         )
+        self.assertEqual(2, len(_result_paths))
```

Comment:
@wesleybl this is really a nitpick nitpick now. If you move this line one up you use the variable right after its declaration. I usually consider it best practice anyways to test the number of results first and then the actual results.

Reply:
Done.